incubator/hnc/internal/version/version.go

```diff
@@ -0,0 +1,19 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+// Version is verion for HNC. This variable is overwritten when build time.
```

Comment:
Maybe a bit more information is useful here: "This variable is overwritten in cloudbuild.yaml to match the version of the Docker image."

Reply:
Yeah! I think it's better. Let me update the comment.